aunt-leahs-app/src/components/TableComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../styles.css';
+import { Table, TableBody, TableCell, Paper, TableHead, TableRow } from '@material-ui/core';
+
+// REQUIRES: data prop has to be array of objects where each object key represents the name of the field in camelcase.
+//					 Refer to data in constants.jsx
+//					 each field in data should be either string or boolean. boolean fields are displayed '✓' or '✗'
+
+const formatHeaderTitle = (title) => {
+	return title.replace(/([A-Z])/g, ' $1').replace(/^./, (str) => str.toUpperCase());
+};
+
+const TableComponent = (props) => {
+	const { data } = props;
```

Comment:
You can move this destructuring to the line above. 

`const TableComponent = ({ data }) => {`